annlite/executor.py

```diff
@@ -9,57 +9,42 @@
 from jina import Executor, requests
 from jina.logging.logger import JinaLogger
 
-from .index import AnnLite
-
 
 class AnnLiteIndexer(Executor):
-    """
-    A simple indexer that wraps the AnnLite indexer and adds a simple interface for indexing and searching.
-    """
-
     def __init__(
         self,
-        n_dim: int = 0,
+        n_dim: int = 128,
```

Comment:
nope, keep it as `0` to enforce the user set this parameter, otherwise, raise error.